server/src/modules/Question/question.service.ts

```diff
@@ -39,26 +39,18 @@ export class QuestionService {
         }
     }
 
-    async insert(question: QuestionDto): Promise<QuestionDto> {
+    async insert(question: QuestionDto): Promise<Question> {
         const newQuestion = new Question();
```

Comment:
const newQuestion = this.questionRepository.create(question);     
.....     
return this.questionRepository.save(newQuestion);       

https://github.com/typeorm/typeorm/blob/master/docs/repository-api.md

Reply:
fixed